pkgs/top-level/all-packages.nix

```diff
@@ -40728,4 +40728,6 @@ with pkgs;
   reindeer = callPackage ../development/tools/reindeer { };
 
   charasay = callPackage ../tools/misc/charasay { };
+
+  please-cli = callPackage ../applications/misc/please-cli { };
```

Comment:
Keep the file roughly alphabetically sorted.

Reply:
Ain't that a little late for that? Scrolling through the last 1000 lines and the first 2000 lines I could only see local ordering (within 100 loc) which, in a file of about 41000 loc is about the same as random order.

As I cannot find a significantly better place than the current one, I will leave it there. If you prefer otherwise, please reference a better place.